langchain4j-parent/pom.xml

```diff
@@ -266,6 +265,13 @@
                 <version>${elastic.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.fasterxml.jackson</groupId>
+                <artifactId>jackson-bom</artifactId>
+                <version>${jackson.version}</version>
+                <type>pom</type>
+            </dependency>
+
             <dependency>
                 <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Now once you've added `jackson-bom`, does it make sense to list all jackson dependencies here?

Reply:
also. if add bom in  `<dependencyManagement>`, i think  need add `<scope> import </scope> `

Reply:
You are both right, I used the import scope and simplified the code